bloom-housing/bloom
# feat: map marker endpoint

This PR addresses [#(889)](https://github.com/metrotranscom/doorway/issues/889)


## Description

Creates a public endpoint to return the id, longitude, and latitude of every open listing called `mapMarkers`.

The existing list endpoint will be used with view params to handle the other endpoint case.

## How Can This Be Tested/Reviewed?

Using OpenAPI or by hitting the API directly through another client, call `listings/mapMarkers`
Verify only open listing id's are returned and check the lat and long returned matches what is in the db.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
